doc/requirements.txt

```diff
@@ -14,6 +14,6 @@ sphinx-autodoc-typehints==1.10.3
 sphinx-copybutton
 sphinx-automodapi
 sphinxcontrib-bibtex==0.4.2
-tensorflow==2.5.1
-thewalrus>=0.16.0
-toml
\ No newline at end of file
+tensorflow==2.6.1
+http://xanadu-wheels.s3-website-us-east-1.amazonaws.com/thewalrus-0.17.0.dev0+20211117203611-cp38-cp38-manylinux2014_x86_64.whl#sha256=f8cef1f05a903eec2b563758b7542490681dfc899e57cce1542385546e222306
```

Comment:
Will be switched to `thewalrus>=0.17.0` when making the version bump PR, after TW has been released.